umm-spec-lib/resources/example_data/dif/C1214586614-SCIOPS.xml

```diff
@@ -10,7 +10,7 @@
       <Dataset_Creator>Rothacher, Markus</Dataset_Creator>
       <Dataset_Title>GPS ground tracking data, Rinex formatted, 10 sec sample rate</Dataset_Title>
       <Dataset_Series_Name>CHAMP Orbit and Gravity Field Products</Dataset_Series_Name>
-      <Dataset_Release_Date>2001 - 2010 </Dataset_Release_Date>
```

Comment:
Then I am OK with the dates since Lauren said that we have a ticket to address it.